google-cloud-spanner/lib/google/cloud/spanner/commit.rb

```diff
@@ -330,6 +286,40 @@ def mutations
 
         protected
 
+        ##
+        # Generates mutations from `rows` to be performed on a given table. It also converts objects for :PROTO types
+        # specified within the table DDL.
+        #
+        # @param [String] table The name of the table in the database to be
+        #   modified.
+        # @param [Array<Hash>] rows One or more hash objects with the hash keys
+        #   matching the table's columns, and the hash values matching the
+        #   table's values.
+        # @param [String] type The type of mutation to be performed.
+        #
+        def mutations_from_rows table, *rows, type
```

Comment:
I think you'll need to remove the splat (asterisk) from `rows`, since you're just passing a normal array.